packages/react-server/core/ReactServerAgent/Request.js

```diff
@@ -71,9 +74,14 @@ Request.prototype.query = function (queryParams) {
 
 Request.prototype.send = function (postParams) {
 	if (typeof postParams === 'undefined') {
-		return merge({}, this._postParams);
+		return merge({}, this._postParams || {});
+	}
+	if (postParams !== null) {
```

Comment:
I 100% agree.  However, it seems that the current implementation of all of these functions does something different if you call it like `.send()` (returns the currently set post parameters) vs `.send({})` (which saves the post parameters and returns `this`).  In my opinion, that is a bad design for all of those functions...trying to combine setters + getters.  That's outside the scope of this PR though.